src/migrations/1631103259350-AddNamehashColumn.ts

```diff
@@ -0,0 +1,25 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddNamehashColumn1631103259350 implements MigrationInterface {
+  name = 'AddNamehashColumn1631103259350';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "role" ADD "namehash" character varying`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "organization" ADD "namehash" character varying`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "application" ADD "namehash" character varying`,
+    );
```

Comment:
Not a big deal, but the `namehash` actually isn't character varying. It is a fixed 256bit (or 64 hex characters, 66 with the `0x` prefix), so the column size could be fixed as well. Not sure if makes sense to do this though 🤷  https://dba.stackexchange.com/a/21496 . So probs best to leave it character varying 🤔 ? what do you think @ahmedolaibrahim ?

Reply:
I think it's probably best to leave it as character varying, I don't think we stand to gain much from doing so.